src/controllers/advanced-search/search.controller.ts

```diff
@@ -1,12 +1,13 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
```

Comment:
I can't see this used anywhere in the class, is this required?

Reply:
Not being used, will remove it